docker-compose.yml

```diff
@@ -112,6 +127,43 @@ services:
       JAVA_OPTS: -Xmx128m
 
 
+## Observability
```

Comment:
This looks useful. However, this example is used for labs on potentially machines with limited resources.

The observability containers need to be optional - e.g. a separate Dockerfile.

Reply:
Done, moved to docker-compose-observable.yml